app/views/template_inputs/_invocation_form.html.erb

```diff
@@ -12,7 +12,7 @@
         :id => input.name,
         :class => input.hidden_value? ? 'masked-input' : '' %>
     <% else %>
-      <%= mount_report_template_input(input_fields.object) %>
+      <%= mount_report_template_input(input_fields.object, local_assigns.fetch(:search_input_options, {})) %>
```

Comment:
I feel that `search_input_options` is a bad name, as it can be any input (currently just Search or DateTime, but we can support more in the future)
Also, do we need the variable passing for that? We have the form object, I feel that we should change this to pass the form name from the form object.

```suggestion
      <%= mount_report_template_input(input_fields.object, input_name: input_fields.name) %>
```
It would of course required some changes in the React components, but I believe that should be done anyway.